packages/web-react/src/components/FileUploader/useFileUploaderInput.ts

```diff
@@ -143,7 +167,14 @@ export const useFileUploaderInput = (props: UseFileUploaderInputProps): UseFileU
     onDrop: onDropHandler,
   });
 
+  useEffect(() => {
+    updateDropZoneVisibility();
+    // We only want a dependency for fileQueue
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [fileQueue]);
```

Comment:
If you are using this dependency, so pass it to the updater -> `updateDropZoneVisibility(fileQueue)`.

Reply:
fixed